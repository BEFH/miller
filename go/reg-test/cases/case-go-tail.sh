run_mlr --opprint tail -n 1 $indir/medium.dkvp
run_mlr --opprint tail -n 1 -g a $indir/medium.dkvp
run_mlr --opprint tail -n 1 -g a,b $indir/medium.dkvp
