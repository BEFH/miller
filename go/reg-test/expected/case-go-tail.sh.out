
mlr --opprint tail -n 1 ./reg-test/input/medium.dkvp
a   b   i  x                   y
pan hat 40 0.30302850095892986 0.2884018711352886

mlr --opprint tail -n 1 -g a ./reg-test/input/medium.dkvp
a   b   i  x                    y
pan hat 40 0.30302850095892986  0.2884018711352886
eks pan 31 0.5701563454882129   0.822178550628332
wye pan 37 0.056537252420374995 0.5450112991603523
zee zee 35 0.12565828887119346  0.1936186210637818
hat pan 39 0.4591095934684415   0.6526252088703994

mlr --opprint tail -n 1 -g a,b ./reg-test/input/medium.dkvp
a   b   i  x                     y
pan pan 16 0.5736853980681922    0.7554169353781729
eks pan 31 0.5701563454882129    0.822178550628332
wye wye 32 0.5112738997714514    0.33107563648490446
eks wye 20 0.38245149780530685   0.4730652428100751
wye pan 37 0.056537252420374995  0.5450112991603523
zee pan 30 0.7514525581952406    0.5869901584461873
eks zee 14 0.5207382318405251    0.34141681118811673
zee wye 22 0.9270813051677103    0.5523236165919244
hat wye 34 0.49061881414751796   0.4721354893261027
pan wye 10 0.5026260055412137    0.9526183602969864
zee eks 17 0.29081949506712723   0.054478717073354166
hat zee 18 0.05727869223575699   0.13343527626645157
hat eks 21 0.0023058336158985338 0.16665492414949412
wye hat 24 0.7286126830627567    0.19441962592638418
pan eks 25 0.291444729043585     0.2680100577363237
eks eks 29 0.05713488327208227   0.45012758797367913
hat hat 28 0.5811879718711469    0.3934172489640808
hat pan 39 0.4591095934684415    0.6526252088703994
zee zee 35 0.12565828887119346   0.1936186210637818
pan hat 40 0.30302850095892986   0.2884018711352886
pan zee 38 0.508250895522605     0.12919877119684198

